src/Tribe/Organizer.php

```diff
@@ -640,7 +640,9 @@ public static function get_fetch_callback( $event ) {
 		}
 
 		return static function () use ( $event ) {
-			$organizer_ids = array_map( 'absint', (array) get_post_meta( $event, '_EventOrganizerID' ) );
+			$organizer_ids = array_filter(
+				array_map( 'absint', (array) get_post_meta( $event, '_EventOrganizerID' ) )
+			);
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 60155ebff048732f58d10245775ca942</sub>